app/test_main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.main import can_access_google_page
+import pytest
+from types import ModuleType
+
+
+@pytest.mark.parametrize("url, internet_connection, expected", [
+    ("https://www.google.com", True, "Accessible"),
```

Comment:
write like 
    (True, True, "Accessible"),
    (False, False, "Not accessible"),
    (False, True, "Not accessible"),